### ISSUE TITLE
manual change of picture

### ISSUE THREAD

**Issue by Rawkush** [syntax-cbpgec/college-website-2#66]
the pictures shown in front page where in websites where images are displayed of events, and changes after a few seconds but picture can't be changes manually

**Comment by s-r-aman** [comment]
Yes! Due to the increasing complexity of the project, we decided to keep it away. We will add this in the future update. Or you can do it and create a pull request.
